apps/src/maze/dirtDrawer.js

```diff
@@ -62,7 +62,7 @@ export default class DirtDrawer extends Drawer {
    * @param {number} val
    * @return {number}
    */
-  spriteIndexForDirt_(val) {
+  static spriteIndexForDirt_(val) {
```

Comment:
It's exciting to see it added to the language! JavaScript feels like it's growing up ... :joy: